app/javascript/css/pages/track-dashboard.css

```diff
@@ -0,0 +1,158 @@
+@import "../styles";
+
+#page-track-dashboard {
+    & .track-nav {
+        --nav-height: 40px;
+
+        @apply bg-white py-12;
+        & .container {
+            @apply flex;
+        }
+
+        & .back {
+            @apply block mr-24;
+            line-height: var(--nav-height);
+        }
+
+        & .c-track-icon {
+            height: var(--nav-height);
+            width: calc(var(--c-track-icon-width-ratio) * var(--nav-height));
+            @apply mr-20;
+        }
+
+        & .title {
+            @apply mr-64;
+            @apply text-16 font-semibold font-mono;
+            line-height: var(--nav-height);
+        }
+        & .tabs {
+            @apply flex;
+
+            & a {
+                line-height: var(--nav-height);
+                @apply block px-16 rounded-100;
+                @apply text-16 text-textBase font-normal;
+
+                &.selected {
+                    @apply bg-veryLightBlue text-black font-semibold;
+                }
+            }
+        }
+    }
+    & .track-header {
+        & .container {
+            @apply flex items-center py-32;
+            height: 190px;
+        }
+        background: linear-gradient(
+            180deg,
+            #e1ebff 0%,
+            rgba(225, 235, 255, 0) 100%
+        );
+        & .c-track-icon {
+            @apply mr-48;
+
+            height: 125px;
+            width: calc(var(--c-track-icon-width-ratio) * 125px);
+        }
+        & h1 {
+            @apply text-h1 mb-8;
+        }
+        & .joined {
+            @apply text-16;
+        }
+    }
+    & article {
+        & .container {
+            @apply flex;
+        }
+
+        & .lhs {
+            @apply flex-grow;
+        }
+        & .rhs {
+            width: 33%;
+            @apply ml-32;
+        }
+    }
+    & section.contributors {
+        box-shadow: 0px 4px 42px 0px rgba(79, 114, 205, 0.15);
+
+        @apply bg-white rounded-8 pt-24 pb-32 px-32 mb-32;
+
+        & header {
+            @apply flex items-center mb-16;
+            & h2 {
+                @apply text-h4 mr-12;
+            }
+            & .count {
+                @apply rounded-100 px-12;
+                @apply text-textLight text-14 font-medium;
+                @apply border-textLight border-1;
+                line-height: 28px;
+            }
+        }
+
+        & .showcase {
+            @apply flex flex-col gap-8 mb-24;
+
+            & a {
+                @apply flex items-center;
+                @apply border-1 border-borderLight rounded-8;
+                @apply px-16 py-8;
+
+                & .img {
+                    height: 48px;
+                    width: 48px;
```

Comment:
Avatar sizing (along with a lot of other icons/images and stuff) -
xs: 24 x 24
sm: 32 x 32
base: 48 x 48
lg: 64 x 64
xl: 96 x 96
2xl: 128 x 128

So this is a base avatar.

Reply:
Cool. I'll probably standardise these once we've got a load of the images in.